components/Footer.tsx

```diff
@@ -17,17 +18,19 @@ const icons = [
   { SVGcomponent: TiktokIcon, altText:"Visit the International Child Art Foundation's Tiktok page", href:"https://www.tiktok.com/@internationalchildartfdn"},
   { SVGcomponent: LinkedinIcon, altText:"Visit the International Child Art Foundation's Linkedin page", href:"https://www.linkedin.com/company/international-child-art-foundation"},
   { SVGcomponent: PinterestIcon, altText:"Visit the International Child Art Foundation's Pinterest page", href:"https://www.pinterest.com/icaf/"},
-
-
 ];
+
 // Custom box shadows added to apply the effects of "shadow-md" with a negative offset-y
 const Footer = () => {
   return (
     <footer className="relative z-10 flex justify-center w-full shadow-[0_-4px_6px_-2px_rgba(0,0,0,0.1)] shadow-[0_-2px_4px_-2px_rgba(0,0,0,0.1)]">
       <div className="grid py-20 grid-col-1 lg:grid-cols-2 justify-center px-2 w-full max-w-7xl">
         <div className="flex justify-center items-center">
-          <div className="flex-initial w-fit pr-2 lg:pr-12">
-            <Image src={ICAF_logo} alt="ICAF logo"></Image>
+          <div className="w-fit pr-2 lg:pr-12 hidden lg:block">
+            <Image src={ICAF_logo_desktop} alt="ICAF logo"></Image>
+          </div>
+          <div className="w-fit pr-2 lg:pr-12 lg:hidden">
+            <Image src={ICAF_logo_mobile} alt="ICAF logo"></Image>
```

Comment:
Since there are no children being supplied to this component, the component can use a self-closing tag:

```
<Image src={ICAF_logo_mobile} alt="ICAF logo" />
```